src/controllers/FrameController.ts

```diff
@@ -410,6 +411,17 @@ export class FrameController {
         return res.assignImageFromUrl(imageFrameId, url).then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
```

Comment:
It is very clear but in this case I would call it `removeImageFrameSource` (unless other suggestion) and passing null as a source. `setImageSource` should probably not be exposed as such.

To WRS: the idea (from @Matthiee) is to expose more general functions from the engine and splitting it on SDK-side, this method should cover `assignImage`, `assignImageFromUrl`, `assignImageVariable` and the missing `removeImageFrameSource` (already implemented function can be refactored later).

Reply:
Like it was done [here](https://github.com/chili-publish/editor-sdk/pull/219/files#diff-7639a5d7174e6ff877f3b1ab6e7304b200dd8b0eb33f5319157d837cff11192eR193) seems to be the best solution imo.